### ISSUE TITLE
MONDO:0957426 autosomal recessive hyper-IgE syndrome

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#7844]
**Mondo term (ID and Label):**
MONDO:0957426 autosomal recessive hyper-IgE syndrome

**Suggested new label:**
If this is, indeed an exact match to Orphanet 641368 , then the pref name should be Autosomal recessive hyper-IgE syndrome due to ZNF341 deficiency
and this would suggest it is the same concept as OMIM 618282
HYPER-IgE SYNDROME 3, AUTOSOMAL RECESSIVE, WITH RECURRENT INFECTIONS (associated with AR mutation in ZNF341)

BUT the Orphanet term maps to multiple OMIM records, https://www.orpha.net/en/disease/detail/641368 which suggests that there may be some error with the Orphanet record either in name or mappings?
It seems like either it should be broader and generic in name for all AR hyper-IgE syndromes, as named in Mondo OR it should be narrow and exactly match 1 and only 1 MIM record. I will send a message via Orphanet's site as well to try and get clarification. 

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**_Optional_: Any additional information (like supporting evidence, PubMed ID, etc.)**

**Comment by sagehrke** [comment]
Dear @kanems,
Thank you for reaching out to the Mondo Disease Ontology, part of the [Monarch Initiative](https://monarchinitiative.org/)!

We have received your ticket and will review it as soon as possible.

If you have any additional information or updates relevant to your inquiry, please add them in a comment on this ticket. Your input is valuable in helping us understand and resolve the issue effectively.

Thank you for your patience and for your contributions to Mondo!

Sincerely,
The Mondo Team

**Comment by twhetzel** [comment]
@yshwetar I didn't see an Assignee for this so I can continue working on the branch and work you've already created.

**Comment by twhetzel** [comment]
Hi @kanems - thank you for submitting this request. After making these changes and adding an xref on MONDO:0957426 to OMIM:618282, our qc checks identified that this term shares this OMIM xref with MONDO:0032654 and after reviewing the terms it appears these two Mondo terms should be merged. Therefore, in this case we plan to merge the Mondo terms, but will need to allow for the waiting period before one of the terms can be obsoleted and merged. 

The proposed merge plan is to [Merge] MONDO:0957426 'autosomal recessive hyper-IgE syndrome' into MONDO:0032654 'hyper-IgE recurrent infection syndrome 3, autosomal recessive'. The term name should follow the OMIM name and all other labels should be represented as synonyms. Do you agree with this plan?

**Comment by kanems** [comment]
Yes, this seems reasonable (both the plan and protocol to allow for obsolete candidate review by the community).

**Comment by nicolevasilevsky** [comment]
I will work on merging these two terms. This should be available in the June release.
